code/game/gamemodes/gang/recaller.dm

```diff
@@ -318,12 +318,14 @@
 /datum/action/innate/gang/tool
 	name = "Personal Gang Tool"
 	desc = "An implanted gang tool that lets you purchase gear"
+	background_icon_state = "
```

Comment:
I do this every time I try to make a tiny change. If anyone has an idea of how to change a button's "color" var lmk.